source/elens.f90

```diff
@@ -7,59 +7,86 @@ module elens
   use floatPrecision
   use crcoall
   use mod_alloc
```

Comment:
`mod_alloc` should not be included in the module header. Neither should `crcoall` strictly speaking. They should only be included in the subroutines where they're used.